test/entry.ts

```diff
@@ -319,7 +327,11 @@ describe('Entry', () => {
       entry.metadata.traceSampled = false;
       entry.data = 'this is a log';
       const json = entry.toStructuredJSON();
-      assert(withinExpectedTimeBoundaries(new Date(json.timestamp!)));
+      assert(
+        withinExpectedTimeBoundaries(
+          new Date(nanosecondsToDate(json.timestamp!))
```

Comment:
nit: I believe you can remove the `new Date`, since `nanosecondsToDate` already returns a date

Reply:
Oh, right - thanks!